pallets/eth-bridge/src/tests/incoming_events_tests.rs

```diff
@@ -0,0 +1,141 @@
+// Copyright 2023 Aventus Network Systems (UK) Ltd.
+
+#![cfg(test)]
+use crate::{eth::generate_send_calldata, mock::*, request::*, *};
+use frame_support::{
+    assert_err, assert_noop, assert_ok, dispatch::DispatchResultWithPostInfo, error::BadOrigin,
+};
+use sp_runtime::{testing::UintAuthorityId, DispatchError};
+pub extern crate alloc;
+use alloc::collections::BTreeSet;
+use sp_avn_common::{event_discovery::EthBridgeEventsFilter, event_types::ValidEvents};
+
+const ROOT_HASH: &str = "30b83f0d722d1d4308ab4660a72dbaf0a7392d5674eca3cd21d57256d42df7a0";
+const REWARDS: &[u8] = b"15043665996000000000";
+const AVG_STAKED: &[u8] = b"9034532443555111110000";
+const PERIOD: &[u8] = b"3";
```

Comment:
Not sure if these are used anywhere in this test. Maybe we can tidy them up in a separate PR.